src/components/BurgerButton.vue

```diff
@@ -105,6 +105,12 @@
         this.$emit('toggle', this.isActive)
       },
     },
+
+    watch: {
+      active (value) {
+        this.isActive = value
+      },
```

Comment:
It would be good to trigger the `toggle` event when the active status changes.
May I suggest something like the following?
```
active (newValue, oldValue) {
    if (newValue !== oldValue) {
        this.isActive = newValue
        this.$emit('toggle', newValue)
    }
}
```

Reply:
good catch! I just pushed that changes